clamav-docker-1.4.yaml

```diff
@@ -0,0 +1,97 @@
+# This fetch method was chosen because clamav-docker separates their shells scripts out in specific directories
+# upstream also doesn't create tags to track any of these changes, they are tracked mostly in mainline but
+# occasionally tracked on their own branches.
+# If version 1.4 is removed or the has is changed then this build will fail requring manual intervention.
+pipeline:
+  # this is only used for tracking the clamav-docker repo
+  - uses: git-checkout
+    with:
+      repository: https://github.com/Cisco-Talos/clamav-docker
+      expected-commit: 658ba84dfb16f0a70ea6588c5da208e36962e678
+      tag: ""
+      branch: main
+  - uses: fetch
+    with:
+      expected-sha256: 843ec84c968506a7447b4ba137fa0875011f3892b67bebea3ed72abdf84a554e
+      uri: https://raw.githubusercontent.com/Cisco-Talos/clamav-docker/refs/heads/main/clamav/1.4/alpine/scripts/clamdcheck.sh
+      extract: false
+  - uses: fetch
+    with:
+      expected-sha256: f99ed46fb8cd72f899709b652276944ebc5c8d0132890db35154e6dae365814e
+      uri: https://raw.githubusercontent.com/Cisco-Talos/clamav-docker/refs/heads/main/clamav/1.4/alpine/scripts/docker-entrypoint-unprivileged.sh
+      extract: false
+  - uses: fetch
+    with:
+      expected-sha256: 4034f6d63ee6c1d1ed3686733b5722f4b19055b623c82843927613f4e2f7c641
+      uri: https://raw.githubusercontent.com/Cisco-Talos/clamav-docker/refs/heads/main/clamav/1.4/alpine/scripts/docker-entrypoint.sh
+      extract: false
+  - runs: |
+      mkdir -p "${{targets.destdir}}"/usr/bin
+      mv docker-entrypoint-unprivileged.sh "${{targets.destdir}}"/usr/bin/init-unprivileged
+      mv docker-entrypoint.sh "${{targets.destdir}}"/usr/bin/init
+      mv clamdcheck.sh "${{targets.destdir}}"/usr/bin/clamdcheck.sh
+      chmod -R 755 "${{targets.destdir}}"/usr/bin/
+
+update:
+  enabled: true
+  git: {}
+  schedule:
+    period: weekly
+    reason: Upstream does not maintain tags or releases
+
+test:
+  environment:
+    contents:
+      packages:
+        - bash
+        - clamav-1.4
+        - clamav-daemon
+        - clamav-db
+        - clamav-doc
+        - clamav-libunrar
+        - clamav-milter
+        - freshclam
+        - libstdc++
+        - netcat-openbsd
+        - tini
+        - tzdata
+    accounts:
+      groups:
+        - groupname: clamav
+          gid: 100
+      users:
+        - username: clamav
+          gid: 100
+          uid: 100
+      run-as: 0
+    environment:
+      ENV TINI_SUBREAPER: 1
+  pipeline:
+    - name: "Failure test: clamdcheck.sh"
+      runs: |
+        bash /usr/bin/clamdcheck.sh && exit 1;
+    # this test will fail locally with run-as set to 0, but pass pipeline.
+    # switch to run-as: 100 if running locally.
+    - name: "Init test"
+      runs: |
+        sed -i 's/^#TCPSocket 3310/TCPSocket 3310/' /etc/clamav/clamd.conf
+        sed -i 's/^#TCPAddr localhost/TCPAddr localhost/' /etc/clamav/clamd.conf
+        sed -i '/^set -eu/a sleep 20' /usr/bin/clamdcheck.sh
```

Comment:
If we need to do some `sed` stuff here, should we have a compat subpackage here that will tackle this? I was wondering the image-side about what to do while using `clamdcheck.sh`.